packages/@uppy/utils/src/getFileTypeExtension.js

```diff
@@ -7,10 +7,13 @@ const mimeToExtensions = {
   'audio/ogg': 'ogg',
   'video/webm': 'webm',
   'audio/webm': 'webm',
+  'video/x-matroska': 'mkv',
   'video/mp4': 'mp4',
   'audio/mp3': 'mp3'
 }
 
 module.exports = function getFileTypeExtension (mimeType) {
+  // Remove the ; bit in 'video/x-matroska;codecs=avc1'
+  mimeType = mimeType.replace(/;.*$/, '')
   return mimeToExtensions[mimeType] || null
```

Comment:
Feels like a case-specific solution, but I guess it’s standard to use `;` in this case, so it’ll work for others as well?

Reply:
yeah, my understanding is that `;` is the standard way to do this sort of thing—eg. in my Chrome, I had the same issue but with `video/webm;codecs=whatever`, and it's common in something like `application/javascript;charset=utf8` too.

Reply:
👍all right then! :)